updating/understanding-openshift-updates.adoc

```diff
@@ -0,0 +1,20 @@
+:_content-type: ASSEMBLY
+[id="understanding-openshift-updates"]
+= Understanding {product-title} updates
+include::modules/common-attributes.adoc[]
+:context: understanding-openshift-updates
+
+toc::[]
+
+With {product-title} 4, you can update a {product-title} cluster with a single click using over the air update.
+Platform administrators are automatically notified when an update is available for their cluster.
+Updates can be applied either from the `oc` command-line interface (CLI) or the web console.
+
+The {product-title} update service builds a graph of upgrade possibilities based on release images in the registry.
+This graph is based on allowable tested update paths from a specific version.
+{product-title} clusters connect to the Red Hat Hybrid Cloud servers and identify which clusters the user is running along with version information.
+Policy engines combine the information from customer entitlement in conjunction with the update graph to tell the cluster which release they can upgrade to.
+Either a cluster administrator or through the automatic update controller edits the Cluster Version Operator (CVO)’s customer resource (CR) with the new version to update to.
```

Comment:
`Either a cluster administrator or an automatic update controller edits ...`?  Or is there a specific automatic update controller we're talking about here?

Reply:
also "CR" is "[custom resource][1]", not "customer resource".

[1]: https://www.bmc.com/blogs/kubernetes-crd-custom-resource-definitions/